test/defs/test_defs_de.rb

```diff
@@ -23,11 +23,11 @@ def test_de
   assert_equal name, (Holidays.on(date, :de, :informal)[0] || {})[:name], "Failed on '#{name}' for date '#{date}'"
 end
 
-[:de_bw, :de_by, :de_st, :de_].each do |r|
+[:de_bw, :de_by, :de_by_aux, :de_st, :de_].each do |r|
```

Comment:
I added this so it tests the holidays of bavaria also valid in augsburg. It doesn't really have todo with holidays/definitions#16